fuse/plugins/micro_physics/lineage_cluster.py

```diff
@@ -407,6 +400,12 @@ def is_lineage_broken(
 
         distance = np.sqrt(np.sum((parent_position - particle_position) ** 2, axis=0))
 
+        if (particle["creaproc"] == "phot") and (particle["edproc"] == "phot"):
```

Comment:
This was necessary to make sure that we do not split a photoionisation peak into multiple parts. This is because there can be some additional ~35keV gamma-phot inside the photoionisation peak ( probably from x-rays )